src/sql/parts/query/editor/queryEditor.ts

```diff
@@ -461,6 +463,16 @@ export class QueryEditor extends BaseEditor {
 		this._estimatedQueryPlanAction = this._instantiationService.createInstance(EstimatedQueryPlanAction, this);
 		this._actualQueryPlanAction = this._instantiationService.createInstance(ActualQueryPlanAction, this);
 
+		this.setTaskbarContent();
+
+		this._configurationService.onDidChangeConfiguration(e => {
```

Comment:
Some places you listen on configuration changes and some places you don't, would it be better to just reload the window when the user selects to enable preview features?

Reply:
You’re right that some places don’t watch the config changes, for example users would have to close and reopen a dashboard after making the change in order to see extension tabs, but I don’t think it’s going to be a super common scenario so I won’t worry about it too much.